automation/test-linux.sh

```diff
@@ -4,18 +4,32 @@ set -ex
 
 template_name=$1
 namespace="kubevirt"
+template_local=""
+template_option=$template_name
+
+if [ -z "$KUBE_CMD" ]
```

Comment:
you don't have to set again `KUBE_CMD` variable. It will be already set from `test.sh` script (which calls this file)

Reply:
Removed